hack/dind-cluster.sh

```diff
@@ -327,7 +327,7 @@ function nodes-are-ready() {
   # Skip the SDN node since nothing is intended to be scheduled on it.
   local node_count=$(${DOCKER_CMD} exec -t "${MASTER_NAME}" bash -c "\
 KUBECONFIG=${DEPLOYED_CONFIG_ROOT}/openshift.local.config/master/admin.kubeconfig \
-oc get nodes | grep -v ${SDN_NODE_NAME} | grep -v NotReady | grep Ready | wc -l")
+oc get nodes | grep -v SchedulingDisabled | grep -v NotReady | grep Ready | wc -l")
```

Comment:
eh, it makes sense after a while :)
(and some of the ugliness is just because the data model is complicated; eg, we have to find the "Ready" condition in the arbitrarily-ordered array of conditions and then check its value)